tests/infrared/16/infrared-openstack.sh

```diff
@@ -21,10 +21,10 @@ OSP_VERSION="${OSP_VERSION:-16.1}"
 OSP_TOPOLOGY="${OSP_TOPOLOGY:-undercloud:1,controller:3,compute:2,ceph:3}"
 OSP_MIRROR="${OSP_MIRROR:-rdu2}"
 LIBVIRT_DISKPOOL="${LIBVIRT_DISKPOOL:-/var/lib/libvirt/images}"
+ENVIRONMENT_TEMPLATE="${ENVIRONMENT_TEMPLATE:-stf-connectors.yaml.template}"
 
 TEMPEST_ONLY="${TEMPEST_ONLY:-false}"
-
-PREFIX="${PREFIX:-osp}"
```

Comment:
Ah yes I recently ran into trouble with this; though I thought it was because I was moving from a PREFIX'd environment to a non-PREFIX'd one. IIRC @pleimer added this so that we could deploy infrared and CRC on the same machine - is this an abandoned feature, or is there a different solution now?

Do we understand what is wrong with the current usage of PREFIX? It seems like it really should work if we've done it righ.

Reply:
I'm not sure what is wrong with the current usage of PREFIX, but it caused all sorts of deployment troubles for me. I had asked the infrared folks about this, and they said in theory it should work, but that they never use it. I suspect there is a bug in some of the Ansible logic or otherwise, and that it's not actually working.

I'd prefer to just see it dropped until someone can spend time doing some additional testing that doesn't cause the script to fail at the first step (clean up).